### ISSUE TITLE
One simple request: either custom or ISO formatted date output

### ISSUE THREAD

**Issue by dwainew** [microsoft/SQL_LogScout#12]
I love the tool, and thanks for your efforts!

However, I always need help understanding it when I see file/folder date/time stamps using something other than yyyyMMdd (ISO) formatting...  The output folder suffix here is one example.  

So first, why ddMMyyyy was chosen? and second...

Could you change this to the ISO standard (which your other parms use)
or
Could you provide a parm (or config) to allow specifying a custom format

Obviously, this could be driven with Powershell, but why make the dev jump through such hoops, just to have consistent file/folder sorting?

**Comment by jpolisel** [comment]
Hi @dwainew, thank you so much for the feedback!

I looked into this, and I understand that the issue is regarding new folder name created using format ".\Output_ddMMyyhhmmss". We will get this addressed in a future release; it should be yyMMddhhmmss instead.

I checked the timestamp of file names and inside of SQL LogScout logs and I couldn't find this issue anywhere else. Did you find the timestamp format to be DMY anywhere else besides the name of the output folder?
